components/popover/popover-mixin.js

```diff
@@ -0,0 +1,99 @@
+import '../colors/colors.js';
+import { css, html } from 'lit';
+
+const isSupported = ('popover' in HTMLElement.prototype);
+
+// eslint-disable-next-line no-console
+console.log('Popover', isSupported);
+
+export const PopoverMixin = superclass => class extends superclass {
+
+	static get properties() {
+		return {
+			/**
+			 * Whether the popover is open or not
+			 * @type {boolean}
+			 */
+			opened: { type: Boolean, reflect: true },
+			_useNativePopover: { type: Boolean, reflect: true, attribute: 'popover' }
```

Comment:
Calling the native `showPopover` or `hidePopover` requires the native `popover` attribute, otherwise an error will be thrown by the browser.